projects/aca-content/src/lib/components/search/search-input-control/search-input-control.component.spec.ts

```diff
@@ -128,4 +139,61 @@ describe('SearchInputControlComponent', () => {
       expect(component.searchFieldFormControl.setValue).not.toHaveBeenCalled();
     }));
   });
+
+  describe('ngOnChanges', () => {
```

Comment:
the rule of thumb is not to test the Angular component lifecycle events. You should be testing the user interactions, not the internal api